.github/pull_request_checklist.md

```diff
@@ -0,0 +1,15 @@
+## Description
+Changes made for this branch
+
+## Checklist
+- [ ] All manual tests passing
+- [ ] All unit tests passing 
+- [ ] All UI tests passing
+- [ ] Code reviewed by peer:
+&nbsp;&nbsp;&nbsp;&nbsp; - [ ] No redundant whitespace
```

Comment:
Why all the non-breaking spaces here? Remove these.

Reply:
@matthewdoreilly was just to indicate that those items are to be checked during 'code review by peer'

Reply:
Ah ok that's fine. How does the markdown look on preview though? Cause it's formatted strange here, https://github.com/sapphireduffy/KainosCareerLattice/blob/2926af4180a17d762783f2fa5e7c2b539d99dcf1/.github/pull_request_checklist.md

Reply:
You could change the pull request template to the below?

```
## Description
Changes made for this branch

## Checklist
- [ ] All manual tests passing
- [ ] All unit tests passing 
- [ ] All UI tests passing
- [ ] Code reviewed by peer
&nbsp; - No redundant whitespace
&nbsp; - Indentation correct
&nbsp; - Naming conventions followed
&nbsp; - Comments added (where needed)
&nbsp; - No commented out redundant code
&nbsp; - Branch name correct
- [ ] Code reviewed by TA
```